git-repo-manager/src/service.rs

```diff
@@ -77,7 +87,7 @@ impl GitRepoManager for GitRepoManagerService {
                 return Err(Status::unknown("Failed cleaning up repository"));
             }
 
-            return Err(Status::unknown("Failed adding post-receive hook"));
+            return Err(Status::unknown("Failed hooks path to post-receive file doesn't exist"));
```

Comment:
Not english, "Failed linking post-receive hook, file does not exist" is a more descriptive message